packages/atomic/cypress/integration/common-assertions.ts

```diff
@@ -10,9 +10,23 @@ export function should(should: boolean) {
   return should ? 'should' : 'should not';
 }
 
-export function assertAccessibility(component: string) {
+export function assertAccessibility(
+  component: string | (() => Cypress.Chainable<JQuery<HTMLElement>>)
+) {
+  const rulesToIgnore = ['landmark-one-main', 'page-has-heading-one', 'region'];
```

Comment:
These three tests are relevant for a whole page, not a single component.

When we test accessibility on a single element by specifying a string as a parameter, they appear to already be skipped, but when passing an element they seem to be tested for some reason.